packages/modelviewer.dev/examples/animation/index.html

```diff
@@ -82,7 +82,7 @@ <h1 class="inner-home">model-viewer</h1>
       <div id="demo-container-1" class="demo"></div>
       <div class="content">
         <div class="wrapper">
-          <a class="lockup" href="../index.html"><div class="icon-button icon-modelviewer-black"></div><h1>examples</h1></a>
+          <!-- <a class="lockup" href="../index.html"><div class="icon-button icon-modelviewer-black"></div><h1>examples</h1></a> -->
```

Comment:
Is this coming back, or should we remove it?

Reply:
should be removed, it was just linking to the home page, which is now in the sidebar,